pagespeed/automatic/proxy_fetch.h

```diff
@@ -342,7 +342,7 @@ class ProxyFetch : public SharedAsyncFetch {
   //
   // This is used only for testing.
   static const int kTestSignalTimeoutMs = 200;
-
+  void SetTrustedInput() { trusted_input_ = true; }
```

Comment:
@jmarantz  re: "nit: style guide says to spell this 'set_trusted_input'."
renamed accordingly